package.monorepo.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Do we need this file?  I think this is a legacy package.json that should be deleted?

Reply:
You are right, I've deleted it.